video/users/client-initialization-rooms/client-initialization.js

```diff
@@ -1,8 +1,7 @@
 $.getJSON('/token', function(data) {
 
   // Create a Video Client and connect to Twilio's backend
-  var videoClient = new Twilio.Video.Client(data.token);
-  videoClient.connect({ to: 'my-room'}).then(function(room) {
+   Twilio.Video.connect({ to: 'my-room'}).then(function(room) {
```

Comment:
Similar comment as above. Also, the `to` parameter changed to `name` (and another nitpick, we should probably regularize the spacing around the curly braces, e.g. `{ foo: 'bar' }` versus `{foo:'bar'}`; i think we should prefer the former).